tests/test_binaries.py

```diff
@@ -146,6 +146,7 @@ def test_eth_regressions(self):
             {'number': 795, 'contract': None},
             {'number': 799, 'contract': 'C'},
             {'number': 807, 'contract': 'C'},
+            {'number': 808, 'contract': 'C'},
```

Comment:
this test is not a regression test, so it doesn't quite belong here (the numbers here correspond to github issues). this seems more like an integration test, so i think that would fit better in a new function for integration tests?